lib/browser/api/net.js

```diff
@@ -62,12 +62,12 @@ class IncomingMessage extends Readable {
       'server',
       'age',
       'expires'
-    ]
+    ])
```

Comment:
Extract this `Set` to the global level so that it's not re-created every time this function is called